.cirrus.yml

```diff
@@ -20,10 +20,10 @@ env:
 #     CIRRUS_SHELL: "bash"
 #     COMPILER: msvc
 #     BUILD_ARTIFACTSTAGINGDIRECTORY: ${CIRRUS_WORKING_DIR}\artifacts\
-#     QT_VER_MSVC: 'msvc2019'
+#     QT_VER_MSVC: 'msvc2022'
 #     QT_DATE: '202310040911'
 #     QTDIR: C:\Qt\${QT_VER}\${QT_VER_MSVC}_64
-#     VULKAN_VER: '1.3.224.1'
```

Comment:
This is intentionally this specific version iirc. It is not tgat we forgot to update.

Reply:
OK I didn't know

Reply:
And neither is this change reflected in the submodules